Example/healthvault-ios-sdk/ViewControllers/Timeline/MHVTimelineSnapshotViewController.m

```diff
@@ -68,9 +68,10 @@ - (void)loadTimeline
     [self.statusLabel showBusy];
     
     MHVLocalDate* startDate = [[MHVLocalDate alloc] initWithObject:@"2017-05-29" objectParameters:nil];
```

Comment:
Maybe update to be the past 60 days or something?  Running the app a couple years from now could be a lot of data